service/initialise.go

```diff
@@ -81,13 +83,34 @@ func (e *ExternalServiceList) GetPGXPool(ctx context.Context, cfg *config.Config
 	return pgxConn, nil
 }
 
+func (e *ExternalServiceList) getRDSDB(ctx context.Context, cfg *config.Config) (api.RDSAreaStore, error) {
+	rds, err := e.Init.DoGetRDSDB(ctx, cfg)
+	if err != nil {
+		log.Error(ctx, "failed to create rds client", err)
```

Comment:
Please can we stop logging errors and then returning them! You end up with the same error logged multiple times, not having an accurate picture of what's going on. Much better to return the error wrapped with the message `return nil, fmt.Errorf("failed to create rds client: %w", err)`